src/js/view/controls/rightclick.js

```diff
@@ -109,7 +109,7 @@ export default class RightClick {
             return;
         }
 
-        removeClass(this.playerElement, 'jw-flag-rightclick-open');
+        removeClass(this.containerElement.parentElement, 'jw-flag-rightclick-open');
```

Comment:
we’re still targeting the same element here I hope? Flag classes are expected to go on the player container.

Reply:
@zetagame ^

Reply:
yeah, the flag is going on .jwplayer